src/tests/sorting.spec.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+
+async function isSortedAlphabetically(array, isDescending = false) {
+    const sortedArray = [...array].sort();
+    if (isDescending) {
+        sortedArray.reverse();
+    }
+    return array.every((value, index) => value === sortedArray[index]);
+}
+
+async function isSortedByPrice(array, isDescending = false) {
+    const sortedArray = [...array].sort((a, b) => (isDescending ? b - a : a - b));
+    return array.every((value, index) => value === sortedArray[index]);
+}
```

Comment:
It is better to move business logic to some helper

Reply:
A few comments ago, Artem wrote "it's better not to have verifications inside of page objects. Better to move it to the test itself for better readability"

Reply:
ok, we can leave it here, but sometimes if we have a bunch of methods, it's better to move it into separate modules or classes depending on the context of the business logic.

Reply:
But it's definitely better to have it **not** inside the PO

Reply:
I will remove it from the tests, but where is the best place to place this file in the project?
for example, I will name the file
**sorting.js**